### ISSUE TITLE
test on in the wild images

### ISSUE THREAD

**Issue by ghost** [Ziyu0/max-sliced-stargan#1]
can I test on my own images for facial expressions attributes?

**Comment by Ziyu0** [comment]
Yes, you can train your own model on specific facial expressions attributes and test it. However, I believe you'll need to write your own data loader if the directory structure of your dataset is different from the CelebA dataset, or if you have different annotations for the attributes. I suggest you refer to the original [StarGAN repo](https://github.com/yunjey/stargan) for how to write your own data loader. If you're considering training on the CelebA dataset, and then test it on your own images, you might not need to write a data loader for it. Just make sure to preprocess your images such that they are in the same form as the training data. You can also consider rewriting the `test` function in `trainer.py` such that it loads your customed images instead of retrieving images from the data loader.

**Comment by ghost** [comment]
@Ziyu0 thanks have you tried training a model with facial attributes like smile frown, I was wondering if you are getting artifacts and if you could share your results and the model
